networkx/algorithms/simple_paths.py

```diff
@@ -106,14 +106,18 @@ def all_simple_paths(G, source, target, cutoff=None):
     target : nodes
        Single node or iterable of nodes at which to end path
 
-    cutoff : integer, optional
-        Depth to stop the search. Only paths of length <= cutoff are returned.
+    cutoff : integer, dict, optional
+       If an integer is passed then specifies the depth to stop the search.
+       Only paths of length <= cutoff are returned.
+       If a dictionary is passed then different cutoffs can be used for different weights.
+       (e.g. {'weight1': cutoff_value1, 'weight2': cutoff_value2})
+       The depth of search can be specified by using `None` as key (e.g. {None: cutoff})
 
     Returns
     -------
     path_generator: generator
        A generator that produces lists of simple paths.  If there are no paths
-       between the source and target within the given cutoff the generator
+       between the source and target within the given cutoff_len the generator
```

Comment:
Did you mean to change this?

Reply:
nope, it's just a remnant from older version